product_docs/docs/pgd/5/durability/index.mdx

```diff
@@ -16,21 +16,21 @@ navigation:
  - administering
  - legacy-sync
  - limitations
-description: Durability options, commit scopes and lag control in EDB Postgres Distributed (PGD) 5.
+description: Durability options, commit scopes, and lag control in EDB Postgres Distributed (PGD) 5.
```

Comment:
@djw-m was wondering why you needed to specify the product name, the full product name, and version number in this description.

Reply:
I don't, I missed it post the LLM description generation.

```suggestion
description: Durability options, commit scopes, and lag control in PGD.
```